lib/toopher_api.rb

```diff
@@ -189,8 +318,25 @@ def parse_request_error(decoded)
   end
 end
 
+# Contains advanced ToopherApi methods associated with Pairings
+class Pairings
+  def initialize(raw)
+    @raw = raw
+  end
+
+  # Check on the status of a previous pairing request
+  #
+  # @param [String] pairing_id The unique string identifier id returned by a previous pairing request.
+  #
+  # @return [Pairing] Information about the pairing request
+  def get_by_id(pairing_id)
+    Pairing.new(@raw.get('pairings/' + pairing_id))
```

Comment:
I wasn't sure which would be best. Like how in python I see we go between `'pairings/{0}'.format(pairing_id)`, `'pairings/%s/' % pairing_id`, and `'pairings/' + pairing_id` everywhere. What do you think is best?